### DISCUSSION TITLE
Endless loading of the "image tracking" example in README

### DISCUSSION THREAD

**Discussion by Sadji0** [AR-js-org/AR.js#284] [Help]
Hi!
I tryed to use example in README as "Image tracking", but it's not wroking. Endless loading and eat much browser memory.
What should I do?
I also tryed to do copy paste and change ways for working in my github webpage. But it's not worked. But marker tracking is working

**Comment by msoula** [comment]
You should display debug console in your browser (F12 on both chrome and firefox) then check whether there is an error when loading assets.
If a-frame script fails, there won't be any visible information but on the console.

**Comment by rainerlonau** [comment]
I had the same experience. The code at https://github.com/AR-js-org/AR.js#-image-tracking doesn´t work out of the box. It´s because line 39 and 46 need to be adjusted.
There´s this hint in the code:
`<!-- we use cors proxy to avoid cross-origin problems ATTENTION! you need to set up your server -->`

The code for the Image Tracking Example here https://ar-js-org.github.io/AR.js-Docs/ worked out of the box.
